api/package.json

```diff
@@ -2,7 +2,6 @@
   "name": "@opentelemetry/api",
   "version": "1.7.0",
   "description": "Public API for OpenTelemetry",
-  "main": "build/src/index.js",
```

Comment:
From what I can tell, `typedoc` crashes because the `package.json#main` is missing. We can leave it here since `package.json#exports` would take precedence over `main`.